pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,42 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from django.core.management.base import NoArgsCommand
 
+from django_redis import get_redis_connection
+from redis import WatchError
```

Comment:
I used the same order as here https://github.com/translate/pootle/blob/master/pootle/core/mixins/treeitem.py#L22-L23